test/topology/pytest.ini

```diff
@@ -4,3 +4,7 @@ asyncio_mode = auto
 log_cli = true
 log_format = %(asctime)s %(levelname)s %(message)s
 log_date_format = %Y-%m-%d %H:%M:%S
+
+markers =
+    medium: tests that take several minutes to run
+    large: tests that take more that ten minutes to run
```

Comment:
Nitpick: Do you really want two different markers "medium" and "large"?
In test/alternator, running more than 600 tests takes less than 30 second, so I considered any test that takes on its own 30 seconds to be "very slow", and added just one marker - "veryslow". I think that a test that take 9 minutes and a test that takes 19 minutes is equally problematic, and don't want to run either of those during development. So I thought that one level of "slow" marker might be enough. But I don't have strong feelings in this matter.

Reply:
I don't have strong feelings here either, we just used `medium` and `large` at my previous job. I'm ok with just `slow`.

Reply:
The two designations "medium" and "large" are fine if you have in mind some scenario where you want to run all "medium" tests but not "large" tests - e.g., run "medium" tests every day but "large" tests every week, or run "medium" tests on every next promotion and "large" tests just once a day, or something like that. I think we should first consider if we have such a plan, and then can come up with tags that fit it.

By the way, ideally, we shouldn't need to have tags that say how long is each test, because the test framework can calculate the amount of time that each test (and here I mean "test" in the pytest sense, i.e., a test functions) takes on average. But we can leave this till much later.

Reply:
The only scenario I have right now is to run several long tests daily instead of every commit, so I'll change these to just "slow".